charts/gardener-dashboard/values.yaml

```diff
@@ -69,6 +69,7 @@ frontendConfig:
       end: 00 08 * * 1,2,3,4,5
     production: ~
   seedCandidateDeterminationStrategy: SameRegion
+  gardenletRespectsSyncPeriodOverwrite: true # this flag should be in sync with gardenlet-configuration controllers.shoot.respectSyncPeriodOverwrite https://github.com/gardener/gardener/blob/045b03f02d7bf47a9ef39e2981ce92b86e444bda/pkg/gardenlet/apis/config/v1alpha1/types.go#L223-L226
```

Comment:
Maybe more gardenlet configuration properties must be added. Rename to `gardenlet.respectSyncPeriodOverwrite`. We should use the same property name.

Reply:
✅